package.json

```diff
@@ -162,9 +162,13 @@
     },
     "lint": {
       "dependencies": [
-        "./packages/core:lint"
+        "lint:components",
```

Comment:
Maybe we can also add linting styles?

Reply:
Good idea! I'd like to do that in a separate PR though. The latest `stylelint` major has some pretty breaking changes.

Reply:
ok, sure :)